autonomy/deploy/generators/docker_compose/base.py

```diff
@@ -161,12 +160,9 @@ def build_agent_config(  # pylint: disable=too-many-arguments,too-many-locals
 
     if dev_mode:
         config += "      - ./persistent_data/benchmarks:/benchmarks:Z\n"
-        config += (
-            "      - ./persistent_data/venvs:/home/ubuntu/.local/share/virtualenvs:Z\n"
-        )
-        config += f"      - {package_dir}:/home/ubuntu/packages:rw\n"
+        config += "      - ./persistent_data/venvs:/root/.local/share/virtualenvs:Z\n"
+        config += f"      - {package_dir}:/root/packages:rw\n"
         config += f"      - {open_aea_dir}:/open-aea\n"
-        config += f"      - {open_autonomy_dir}:/open-autonomy\n"
```

Comment:
We don't require open-autonomy in editable mode

Reply:
How can we use a local, under-development version?

Reply:
The open-autonomy is not used by agent runtime so there's no point in having a editable installation